content/agent/proxy.md

```diff
@@ -444,6 +444,97 @@ To verify that everything is working properly, review the HAProxy statistics at
 {{% /tab %}}
 {{< /tabs >}}
 
+## Using NGINX as a Proxy
+
+[NGINX][3] is a web server which can also be used as a reverse proxy, load balancer, mail proxy, and HTTP cache. You can use NGINX as a proxy to send logs to Datadog:
+
+`agent ---> nginx ---> Datadog` or  
+`external log shipper ---> nginx ---> Datadog`
```

Comment:
Let's remove this one and focus on the use case were the agent is used (especially as the endpoint is different when not using the agent).